star_wars_magic_methods/people.py

```diff
@@ -1,3 +1,73 @@
 class People(object):
-    def __init__(self, name, dark_side=False):
-        pass
+    def __init__(self, name, dark_side=False, light_side=True):
+        self.name = name
+        self.dark_side = dark_side
+        self.light_side = light_side
+        
+        if self.dark_side == True:
+            self.light_side = False
```

Comment:
alternatively, can explore to use `not` operator

`self.light_side = not self.dark_side`